android/app/src/main/res/values/strings.xml

```diff
@@ -62,4 +62,9 @@
     <string name="location_service_info_title">Do you want to enable location service?</string>
     <string name="ask_permission">Ask permission</string>
     <string name="updated">"Updated"</string>
+    <string name="device_mode">Use device mode</string>
```

Comment:
This one seems to be missing fi/sv translations

Reply:
Are these translations OK?:
- Käytä laitetilaa
- Använda enhetsläget

Reply:
In Figma the english translation is "Use default". iOS widgets use that one and then

fi: Oletus
sv: Använd standard (suggested by ChatGPT)

Reply:
Well, OK, I use that too now.
Changed.